code/modules/mining/aux_base_camera.dm

```diff
@@ -220,32 +212,6 @@
 		return
 	B.RCD.toggle_window_glass()
 
-/datum/action/innate/aux_base/place_fan
```

Comment:
are you going to replace this with anything?

Reply:
aren't there doors?